### PR TITLE
Removed some Dom errors

### PR DESCRIPTION
**Pull request by TOLoneWolf**
Removed some Dom errors:
- active
- selectedMods
- selectedItems

Based on:
<https://github.com/styled-components/styled-components/issues/1198#issuecomment-425650423>

### CODE REVIEW

File: src/common/modals/InstanceManager/Mods.js [hunk 1]
```diff
@@ -539,20 +556,7 @@ const Mods = ({ instanceName }) => {
...
-                `&:hover {
-                cursor: pointer;
-                path {
-                  cursor: pointer;
-                  transition: all 0.1s ease-in-out;
-                  color: ${props.theme.palette.error.main};
-                }
-              }`}
-            `}
+            selectedMods={(selectedMods.length > 0).toString()}
```
Comment (blarfoon) [inline src/common/modals/InstanceManager/Mods.js]: Please don't do this. It's enough to remove the key from the dom element

File: src/common/modals/InstanceManager/Notes.js [hunk 2]
```diff
@@ -167,7 +167,7 @@ const BlockButton = ({ format, icon }) => {
         margin: 0 2px;
         border: ${props => `solid 2px ${props.theme.palette.primary.main}`};
       `}
-      active={isBlockActive(editor, format)}
+      active={isBlockActive(editor, format).toString()}
```
Comment (blarfoon) [inline src/common/modals/InstanceManager/Notes.js]: Same here, you should keep it as boolean but not passing it to the dom element

**Review by blarfoon — CHANGES_REQUESTED** [review]

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ src/common/modals/InstanceManager/Notes.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5477610&bid=17952050)

File: src/common/modals/InstanceManager/Notes.js (hunk 2 above)
Comment (blarfoon) [inline src/common/modals/InstanceManager/Notes.js]: Same here, remove toString and just strip this down from the props

**Review by blarfoon — CHANGES_REQUESTED** [review]

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ src/common/modals/InstanceManager/Notes.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5477610&bid=17952050)

**Comment by blarfoon** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ src/common/modals/InstanceManager/Notes.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5477610&bid=17952050)

**Pull request merged**